README.md

```diff
@@ -55,14 +55,19 @@ developer's workstation without having to `ssh` directly into the ADB virtual ma
 
 ## Available Commands <a name="commands"></a>
 
-The following table lists the available commands for the plugin and their explanation:
-
-Commands                                                   | Explanations
------------------------------------------------------------|-----------------------------------------
-`vagrant service-manager env`                              | Displays connection information for all active providers in the box.
-`vagrant service-manager env docker`                       | Displays connection information for the Docker provider.
-`vagrant service-manager env openshift` [--script-readable]| Displays connection information for the OpenShift provider. This is optionally available in script readable format too.
-`vagrant service-manager box version` [--script-readable]  | Displays the version and release of the running Vagrant box. This is optionally available in script readable format too.
+The following section lists the available commands for the plugin and their explanation:
+
+1. `vagrant service-manager env [service] [--script-readable]` - Displays connection information for all active services in the box in a manner that can be evaluated in a shell. If a service is specified only the information for that service is displayed. When `--script-readable` is specified the output is in key=value format. Supported services are: Docker; OpenShift
```

Comment:
1. Probably if you put service in quote here that would be more readable.
```
.... If a service(should as `service`) is specified only the information for ..
```

2. I think Supported services should be in comma.